### DISCUSSION TITLE
Regarding Anytest

### DISCUSSION THREAD

**Discussion by xingewh** [Acly/krita-ai-diffusion#897] [Ideas]
First of all, I would like to thank Acly for bringing us such a useful and convenient Krita AI. 
It is extremely practical and easy to use. 
Recently, I tried the Anytest SDXL control model and found it to be very effective. It integrates many functions of different ControlNet models, allowing for precise image control effortlessly. 
I wonder if Acly would be interested in incorporating it into Krita AI.

Author's homepage:
https://note.com/kokekokkosann/n/n46b3967fe895

Model download link:
https://huggingface.co/2vXpSwA7/iroiro-lora/tree/main/test_controlnet2

Reference video: https://www.youtube.com/watch?v=tu3F81T93vA

**Comment by Acly** [comment]
Similar idea here: https://github.com/xinsir6/ControlNetPlus

Need to compare which one performs better.

**Comment by ushimogu** [comment]
First of all, thank you for this amazing and innovative tool.

I used stable diffusion to draw manga with illustrious models.

Until now, inpaint was mainly done with a workflow using Anytest from Controlnet. Anytest is a versatile and very high-performance model, so I commented to let you know that I hope Anytest will be implemented in this tool.

In addition, a method that I think is effective in the current environment is to draw a shape in gray on the input image and output an object of that shape. Since the color is not specified in the image, it is possible to obtain coloring results that match the image, and the prompts seem to follow well. With normal i2i, the color of the input image tends to be overly influenced, and adjustments are severe, so multiple tries are often required.
Even with the previous control net, it is possible to do something similar by referencing only the shape added as a preprocessor, but Anytest has higher performance and often produces the expected results.

Finally, I will continue to use this tool, so I am deeply grateful for your activities and support you.

**Comment by krigeta** [comment]
hello, is there any update about Anytest Implementation?
